lib_runtime/mirage_runtime.ml

```diff
@@ -16,8 +16,12 @@
 
 open Cmdliner
 
-let ocaml_section = "OCAML RUNTIME PARAMETERS"
-let unikernel_section = "UNIKERNEL PARAMETERS"
+(* These argument sections appear in the manpage in reverse alphabetic order -- as ordered here. *)
```

Comment:
```suggestion
```

Reply:
I swear I edited that comment :/ must have lost the commit when rebasing somehow...

Reply:
Ah it was uncommitted and now part of dc796e36c4b67381525123237762a512555e5616